ios/Exponent/Kernel/AppLoader/EXFileDownloader.m

```diff
@@ -1,6 +1,7 @@
 // Copyright 2015-present 650 Industries. All rights reserved.
 
 #import "EXEnvironment.h"
+#import "EXHomeModule.h"
```

Comment:
would be great to put these methods in a different module like `EXProvisioningProfile` so you don't have to import EXHomeModule here.